src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_NUMBER = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
The instance of 'ColorSupplier' should be final. You should declare it as a constant.